### DISCUSSION TITLE
Could anyone help recover my account?

### DISCUSSION THREAD

**Discussion by xyc4en** [community/community#121223] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In early March, my last GitHub account was flagged, prompting me to contact GitHub Support. After a few days, my account was suspended. A month later, I received an email explaining the reason for the suspension:

> Some of the activity on your account has caught the attention of our abuse-detecting systems, including uploading compiled executable files to repositories with no accompanying source code. This has caused a number of hits on tools that identify potentially malicious code.

> 
> Since this activity isn't in keeping with our Terms of Service, we'll have to keep the account restrictions in place.

I believe one reason for this action was the uploading of a zip file containing legacy software to one of my repositories. This software was inherited from the previous maintainer during my work with the school student community and has been in use for a considerable period (about 10+ years). I am unable to locate the corresponding source code, but I'm confident the software is safe based on my antivirus tests. Apart from this, I am unaware of any other reasons for the account's suspension.

I asked if the account could be restored by deleting the repository in question, but after several email exchanges, GitHub Support reiterated that my account would not be recovered, and further correspondence would not be entertained:

> As stated earlier, we'll not be removing restrictions from this account. Any further similar request may be closed without a response.

At the beginning of April, I initiated another ticket to GitHub Support under my current account to appeal, but as of the end of April, I have not received a reply.

The suspended account has been with me throughout my college career and holds profound meaning to me. There is important data and code in my repository that I did not back up locally, and I urgently need access to it. If my account cannot be restored, I respectfully appeal to my community friends for their kind assistance in finding ways to retrieve my code and data.

**Comment by paulospx** [comment]
Hi xych1n,

I can relate to your frustration, as I've encountered a similar issue with GitHub. My account was blocked a few months ago, and despite repeated inquiries, GitHub has not provided a single reason for the suspension. I've wondered if it might be related to having a Ukrainian flag on my profile, but there's been no confirmation on that front. This lack of transparency and inability to back up or delete my account seems to be a violation of GDPR rights. I'm considering legal action as a last resort.

It's particularly challenging because this situation has significantly hindered my ability to work and access important materials. If you find a way to retrieve your data or if your community friends can offer any assistance, I would appreciate hearing about any solutions that might also apply to my situation.

Best,
Paulo

**Comment by ebndev** [comment]
Hi @xych1n,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
